dev/PushNotifications/PushNotificationChannel.h

```diff
@@ -18,6 +19,8 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
 
     private:
         const winrt::Windows::Networking::PushNotifications::PushNotificationChannel m_channel{ nullptr };
+        winrt::Windows::Foundation::DateTime m_Extime{};
```

Comment:
Avoid abbreviations like ext in your code. This is m_expirationTime at the end of the day.